code/espurna/config/hardware.h

```diff
@@ -3335,6 +3335,35 @@
     #define NETBIOS_SUPPORT     1
     #define SSDP_SUPPORT        1
 
+// -----------------------------------------------------------------------------
+// This configuration is for the second generation of devices sold by OBI.
+// https://www.obi.de/hausfunksteuerung/wifi-stecker-schuko-weiss/p/4077806
+// -----------------------------------------------------------------------------
+#elif defined(OBI_OBI_V2)
```

Comment:
Any reason why not name it the same as previous one?
`OBI_WIFI_STECKER_SCHUKO_V2`?

Reply:
No, I was just not aware of the brand behind the device. You're absolutely right this should be renamed (and probalby be relocated so that both are close together? What do you think?)